gradle/libs.versions.toml

```diff
@@ -2,7 +2,19 @@
 kotlin = "1.9.0"
 
 [libraries]
+# Decompilers
+fernflower = { module = "net.fabricmc:fabric-fernflower", version = "2.0.0" }
+cfr = { module = "net.fabricmc:cfr", version = "0.2.1" }
+
+# Runtime depedencies
+mixin-compile-extenions = { module = "net.fabricmc:fabric-mixin-compile-extensions", version = "0.6.0" }
```

Comment:
Should be `-extensions` (won't make a suggestion as it's used in quite a few places)

Reply:
Thanks :)